homeassistant/components/serial/sensor.py

```diff
@@ -65,26 +64,42 @@ def __init__(self, name, port, baudrate, value_template):
 
     async def serial_read(self, device, rate, **kwargs):
         """Read the data from the port."""
-        reader, _ = await serial_asyncio.open_serial_connection(
-            url=device, baudrate=rate, **kwargs
-        )
+        loggedError = False
```

Comment:
Please use snake case. 
```suggestion
        logged_error = False
```